pre/remap_restart/remap_catchANDcn.py

```diff
@@ -52,25 +54,36 @@ def remap(self):
      out_wemin  = config['output']['surface']['wemin']
      surflay    = config['output']['surface']['surflay']
      in_tilefile = config['input']['surface']['catch_tilefile']
+
+     # determine NPE based on *approximate* number of input and output tiles
+      
      if not in_tilefile :
+        if not in_bcsdir:
```

Comment:
@weiyuan-jiang, does the "if" here check for the existence of the file, or just that there's a non-empty variable?  If it's the latter, someone could specify dummy values for `in_tilefile` and `in_bcsdir` and we would get past this check but still wouldn't be able to find out anything about `in_tilenum`    (sorry I don't know enough python...)

Reply:
basically the two 'if' make sure that one of them should not be empty. It is just a check